src/libs/events.js

```diff
@@ -63,4 +65,48 @@ export const PageViewReporter = () => {
   return null
 }
 
+export const captureAppcuesEvent = (eventName, event) => {
+  // Only record "public-facing events" (and related properties) as documented by Appcues: https://docs.appcues.com/article/301-client-side-events-reference
+  const publicEvents = [
+    'flow_started',
+    'flow_completed',
+    'flow_skipped',
+    'flow_aborted',
+    'step_started',
+    'step_completed',
+    'step_skipped',
+    'step_aborted',
+    'step_interacted',
+    'form_submitted',
+    'form_field_submitted'
+  ]
+  if (_.includes(eventName, publicEvents)) {
```

Comment:
TOL curious why the lodash `_.includes` over the regular array `includes`?

Reply:
Habit, I guess. 🤷

Reply:
The lodash version is guarded against the array being undefined or not an array somehow, so we prefer it as best practice.